### ISSUE TITLE
Fire events on (auto-)scrolling?

### ISSUE THREAD

**Issue by chmanie** [taye/interact.js#38]
I have a problem with autoScrolling. When I drag an interactable near the edges of the autoScroll container, the container is scrolled properly but the 'move' event is not fired on the element. So the element stays where it is instead of being dragged while sticked to the mouse pointer.

The same effect appears while manually scrolling when dragging an element, but I did not expect this to work.

So basically the mouse position should be calculated based on the scrolling container instead of the absolute position on the screen. Is that possible?

Setting the deltaSource property seems to have no effect at all for me.

**Comment by zeeshanjan82** [comment]
Hi @taye I am facing similar issue as is mentioned above.
but the workaround does not seem to get fired or solve
When I drag a div and go outside the autoscroll container then the div does not follow the mouse pointed even though the container is scrolling

**Comment by scottfwalter** [comment]
I can't seem to get the scroll event to fire in 1.2.6.  Is my setup incorrect?

            interact(SchedulerDirectiveService.ACTIVITY_CLASS_DOT + ',' + SchedulerDirectiveService.SHIFT_CLASS_DOT)
                .draggable({
                    onstart: dragStartListener,
                    onmove: dragOrMoveListener,
                    onend: dragEndListener,
                    autoScroll: {
                        container: $('.schedule-details')[0],
                        margin: 50,
                        distance: 5,
                        interval: 10
                    },
                    deltaSource: 'client',
                    snap: {
                        targets: [ $window.interact.createSnapGrid({
                            x: (cellWidth / SchedulerDirectiveService.SNAPS_PER_HOUR)
                        }) ]
                    }
                })
                .on('scroll', function () {
                    console.log('SCROLLING');
                });

the console message never seems to happen.  So I'm guessing I don't have my interactable setup properly.  Any ideas?

**Comment by taye** [comment]
@scottfwalter interact.js doesn't fire `scroll` events when autoScroll happens.

**Comment by mila76** [comment]
so don't exist a fix for this problem at all??
